src/Enum.php

```diff
@@ -6,17 +6,19 @@
 class Enum
```

Comment:
Can you also rename this class to BackedEnum as it now only returns backedEnums?

Reply:
Changed it!